state-sync/storage-service/server/src/optimistic_fetch.rs

```diff
@@ -127,7 +127,7 @@ impl DataSubscriptionRequest {
             DataRequest::GetNewTransactionOutputsWithProof(request) => request.known_version,
             DataRequest::GetNewTransactionsWithProof(request) => request.known_version,
             DataRequest::GetNewTransactionsOrOutputsWithProof(request) => request.known_version,
-            request => unreachable!("Unexpected subscription request: {:?}", request),
+            request => unreachable!("Unexpected optimistic fetch request: {:?}", request),
```

Comment:
nit this string can be a const since its used a couple places

Reply:
Maybe... I think it's used 4 times? But the rust compiler is very very smart. I've given up trying to outsmart it 😄